resources/privacy-and-gdpr.md

```diff
@@ -77,38 +108,51 @@ MobileCore.setPrivacyStatus(MobilePrivacyStatus.OPT_OUT);
 #### Syntax
 
 ```swift
-static func setPrivacyStatus(_ status: PrivacyStatus)
+static func update(with consents: [String: Any])
 ```
 
 #### Example
 
 ```swift
-MobileCore.setPrivacyStatus(PrivacyStatus.optedIn)
+// example 1, updating users collect consent to 'yes'
+let collectConsent = ["collect": ["val": "y"]]
+let currentConsents = ["consents": collectConsent]
```

Comment:
can be one liner

Reply:
Just because it can be a one-liner doesn't mean it should be ;) 
It's less clear IMO, leaving it as is.

Reply:
Also, I'm considering making the Obj-c example split into two lines. It's tough to read.

Reply:
sounds good, let's make all platforms consistent then.

Reply:
how do you feel about the current android snippet? I don't think swift -> java needs to be 1:1 as the languages have different features. The current Java code has the nested creation of the dicts. 

What approach do we want to suggest customers to use in Java?